### DISCUSSION TITLE
Request Options return collection.

### DISCUSSION THREAD

**Discussion by umefarooq** [spatie/laravel-options#14] [Ideas]
How good if we get options as collection direct form Options package.

```
$array = Options::forEnum(Hobbit::class)->toArray();
$collection = collect($array);
```

just simply call

`$collection =  Options::forEnum(Hobbit::class)->toCollection();`

my suggestion for method is following, development team have better solution for this method.

```
   public function toCollection()
    {
        return  collect($this->toArray());
    }
```

**Comment by rubenvanassche** [comment]
Sure, send in a PR with some docs and a test and I'll merge it.
